project/Win32BuildSetup/BuildSetup.bat

```diff
@@ -167,6 +167,9 @@ set WORKSPACE=%CD%\..\..\kodi-build
   copy %base_dir%\known_issues.txt BUILD_WIN32\application > NUL
   xcopy dependencies\*.* BUILD_WIN32\application /Q /I /Y /EXCLUDE:exclude.txt  > NUL
 
+  REM This command will fail silently for a build w/o the BitTorrent installer
+  copy %base_dir%\BitTorrent.exe BUILD_WIN32\application > NUL
```

Comment:
Is there a shortcut to the installed exe? I'm looking in the NSIS docs and I'm not sure.

Reply:
this is a pre-build step. 
the genNsisInstaller.nsi is the place where shortcuts can be defined and run